### PR TITLE
Update local ingresses on dev

### PR DESCRIPTION
**Pull request by dhirajshettymoj**
Update local ingresses on dev to remove references to staff facing application and keep only public facing application URL.

### CODE REVIEW

File: helm_deploy/send-legal-mail-to-prisons/values.yaml [hunk 1]
```diff
@@ -10,7 +10,14 @@ generic-service:
...
 
   ingress:
-    enabled: false
+    enabled: true
+    host: app-hostname.local    # override per environment
+    tlsSecretName: send-legal-mail-cert
+    annotations:
+      nginx.ingress.kubernetes.io/configuration-snippet: |
+        location = / {
+          return 301 $scheme://$host/barcode/find-recipient;
```
Comment (dhirajshettymoj) [inline helm_deploy/send-legal-mail-to-prisons/values.yaml:19]: Copied from existing ingress. Redirects to /barcode/find-recipient if the user lands up on /

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: helm_deploy/send-legal-mail-to-prisons/values.yaml (hunk 1 above)
Comment (tpmcgowan) [reply helm_deploy/send-legal-mail-to-prisons/values.yaml:19]: Could put this as a redirect in the Node app itself? Perhaps might need to wait until the unneeded 'other app' code is stripped out (which I guess is why it was configured here in the first place...)

**Review by tpmcgowan — APPROVED** [review]
LGTM

**Pull request merged**